vscode/webviews/components/EnhancedContextSettings.tsx

```diff
@@ -139,14 +140,23 @@ const CompactProviderComponent: React.FunctionComponent<{
     name: string
     inclusion: 'auto' | 'manual'
     handleRemove: (id: string) => void
-}> = ({ id, name, inclusion, handleRemove }): React.ReactNode => {
+    isIgnored: boolean
+}> = ({ id, name, inclusion, handleRemove, isIgnored }): React.ReactNode => {
```

Comment:
Nit not related to changes in this PR: we can drop the return type since `React.FunctionComponent` includes it.

```suggestion
}> = ({ id, name, inclusion, handleRemove, isIgnored }) => {
```